tests/network/expose.go

```diff
@@ -48,16 +47,6 @@ func newLabeledVMI(label string, virtClient kubecli.KubevirtClient, createVMI bo
 		libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding(ports...)),
 		libvmi.WithNetwork(v1.DefaultPodNetwork()))
 	vmi.Labels = map[string]string{"expose": label}
-
-	var err error
-	if createVMI {
-		vmi, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
-		Expect(err).ToNot(HaveOccurred())
-		tests.WaitForSuccessfulVMIStartIgnoreWarnings(vmi)
-		vmi, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Get(vmi.ObjectMeta.Name, &k8smetav1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
-		tests.WaitUntilVMIReady(vmi, libnet.WithIPv6(console.LoginToCirros))
```

Comment:
` tests.RunVMIAndExpectLaunch` doesn't contain this part. It means ip6 won't be configured.

EDIT: `tests.GenerateHelloWorldServer` has the login and ipv6 configuration part so I guess it is ok...

Reply:
miracles can happen...